api/src/v2/Registration/Uploader/OrgDeputyshipUploader.php

```diff
@@ -137,6 +137,15 @@ private function handleNamedDeputy(OrgDeputyshipDto $dto)
 
                 $this->updated['named_deputies'][] = $namedDeputy->getId();
             }
+
+            if ($namedDeputy->emailHasChanged($dto)) {
+                $namedDeputy->setEmail1($dto->getDeputyEmail());
```

Comment:
I know Sirius have had issues with case managers stuffing multiple email addresses into a single field, so might be worth validating the email format before adding it.